modules/core/package.json

```diff
@@ -85,6 +85,7 @@
     "@types/node": "^11.11.4",
     "@types/sinon": "^7.0.6",
     "@types/stellar-sdk": "^0.11.1",
+    "@types/superagent": "^4.1.3",
```

Comment:
seems odd that ```superagent``` wasn't already here?

Reply:
This is just for the types package - superagent is already declared in the dependencies section